perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +76,93 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
+[global]
+ioengine=libaio
+invalidate=1
+blocksize=4k
+direct=1
+runtime=10m
+time_based
+filename={{filename}}
+do_verify=0
+verify_fatal=0
+randrepeat=0
+size={{size}}
+
+{% for rwkind in ('randread', 'randwrite') %}
+{% for iodepth in iodepths %}
+[{{rwkind}}-io-depth-{{iodepth}}]
+stonewall
+rw={{rwkind}}
+iodepth={{iodepth}}
+
+{% endfor %}
+{% endfor %}
+"""
+
+
+def GetIODepths(io_depths):
+  """Parse the io_depths parameter.
+
+  Args:
+    io_depths: a string in the format of the --io_depths flag.
+
+  Returns:
+    An iterable of integers.
+
+  Raises:
+    ValueError if FLAGS.io_depths doesn't follow a format it recognizes.
+  """
+
+  match = IODEPTHS_REGEXP.match(io_depths)
+
+  if match.group(2) is None:
+    return [int(match.group(1))]
+  else:
+    return range(int(match.group(1)), int(match.group(3)) + 1)
+
+
+def WriteJobFile(mount_point):
```

Comment:
To the extent you can, prefer taking arguments to accessing the FLAGS global - it makes testing easier.

The naming is also a bit confusing...this doesn't actually write the fio job.